Tools/Migration/rodinia-nw-dpct/LICENSE.txt

```diff
@@ -0,0 +1,87 @@
+Copyright Intel Corporation
```

Comment:
Is there a reason that you are not using the standard oneAPI License file for the Intel aspect? If not, you should be using this one: https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt

To keep with consistent formatting match the case, please

Reply:
It's the MIT license file. It's just been word-wrapped so it's easier to read and we've added the SPDX line and the link referencing the official MIT license file.

Also, this sample is derived from an existing open-source project, thus the reason there is the original license file from that original project, as well as the MIT license file that the samples are using.